test/e2e/framework/util.go

```diff
@@ -128,6 +129,8 @@ const (
 
 	// SnapshotDeleteTimeout is how long for snapshot to delete snapshotContent.
 	SnapshotDeleteTimeout = 5 * time.Minute
+
+	ControlPlaneLabel = "node-role.kubernetes.io/control-plane"
```

Comment:
IIRC, this label is a kubeadm-ism and we cannot assume all k8s deployers apply it
https://github.com/kubernetes/enhancements/blob/62cad7da88a76ae416e749311a59a91ec4a0bab5/keps/sig-architecture/1143-node-role-labels/README.md?plain=1

if this is for our internal testing with kubeadm, kops we can assume it. but not for the wider conformance.

Reply:
@neolit123 ack, will add a explicit comment that this is for `kops` and `kubeadm` based clusters.